commands/command_push.go

```diff
@@ -44,15 +45,31 @@ func pushCommand(cmd *cobra.Command, args []string) {
 	}
 
 	ctx := newUploadContext(pushDryRun)
+
+	var argList []string
+	if useStdin {
+		scanner := bufio.NewScanner(os.Stdin) // line-delimited
+		for scanner.Scan() {
+			line := scanner.Text()
+			if line != "" {
+				argList = append(argList, line)
+			}
+		}
+		if err := scanner.Err(); err != nil {
+			ExitWithError(errors.Wrap(err, tr.Tr.Get("Error reading from stdin:")))
+		}
```

Comment:
Can we produce an error here if there are other command-line arguments, since we'll be ignoring them?

Reply:
Done 😃